docs/config/template/index.template.html

```diff
@@ -149,6 +149,10 @@ <h2 class="md-toolbar-item md-breadcrumb md-headline" tabindex="-1">
         </div>
       </div>
 
+      <div class="eol-notice">
+          AngularJS Material is now in Long Term Support (LTS) mode. Explore End Of Life (EOL) options <a href="https://blog.angular.io/finding-a-path-forward-with-angularjs-7e186fdd4429">here</a>.
```

Comment:
The `Long Term Support (LTS) mode` should probably link to `/#long-term-support`.

Reply:
done